lib/recognize-stream.ts

```diff
@@ -217,14 +220,19 @@ class RecognizeStream extends Duplex {
 
     const self = this;
 
-    // node params: requestUrl, protocols, origin, headers, extraRequestOptions
+    // node params: requestUrl, protocols, origin, headers, extraRequestOptions, clientConfig options
     // browser params: requestUrl, protocols (all others ignored)
+
+    // for the last argument, `tlsOptions` gets passed to Node's `http` library,
+    // which allows us to pass a rejectUnauthorized option
+    // for disabling SSL verification (for ICP)
     const socket = (this.socket = new w3cWebSocket(
       url,
       null,
       null,
       options.headers,
-      null
+      null,
+      options.rejectUnauthorized ? { tlsOptions: { rejectUnauthorized: false }} : null
```

Comment:
Looks like the ternary expressions need to be swapped. Right now we're setting `rejectUnauthorized` to false if it's true.

Reply:
Oh shoot I did something silly. I'm not supposed to be setting `rejectUnauthorized` at all. I'll fix that now.